src/emcpy/plots/map_plots.py

```diff
@@ -0,0 +1,81 @@
+class MapScatter:
+
+    plottype = 'map_scatter'
```

Comment:
does this need a self?

Reply:
There is no `self` in the class attribute and you can access it as `MapScatter.plottype`
Not sure what is intended here.

Reply:
Most likely @kevindougherty-noaa is trying to treat this as a `global` variable in the class.

Reply:
Yeah I guess that's what I'm asking, I guess the question is when/how is `plottype` used down the road.

Reply:
You know what, I think I confused myself reading class docs and that it doesn't really matter if it is a class variable or an instance variable. However, for it's purpose in the CreateMap class that I have in PR #29, it needs that variable to determine which function to call when plotting on the map.

Reply:
In that case why not set it in the `__init__` instead of making it an instance variable.

Reply:
I'll revert it back to what I had in the first place.